pkg/controller/replication_controller.go

```diff
@@ -204,6 +204,10 @@ func (rm *ReplicationManager) getPodControllers(pod *api.Pod) *api.ReplicationCo
 // When a pod is created, enqueue the controller that manages it and update it's expectations.
 func (rm *ReplicationManager) addPod(obj interface{}) {
 	pod := obj.(*api.Pod)
+	if pod.DeletionTimestamp != nil {
```

Comment:
@bprashanth I worry I don't understand the replication controller (as it is now) to debug what I've added.  A few questions:

1. Are there known scenarios today where fast rescale triggers the RC loop to forget a pod?
2. Do I even need these notifications, or would it be more accurate to simply let the controller handle it now that I have the check in filterActivePods?